src/Foundation/Exception/Handler.php

```diff
@@ -38,10 +38,10 @@ class Handler extends ExceptionHandler
      *
      * This is a great spot to send exceptions to Sentry, Bugsnag, etc.
      *
-     * @param  \Exception  $exception
+     * @param  \Throwable  $exception
```

Comment:
We should be able to get rid of the exception import at the top of the file, and we'll need to update the remaining references to \Exception (especially in the event docblocks) in this class.

Reply:
Exception and Throwable imports are gone now, slashes are passed around like a grandma handing out candy on halloween.

Reply:
It's used here. A lone usage. Do you want me to return the imports so the slashes can go? https://github.com/wintercms/storm/blob/e81f03b9b3450a6dc824c6bc6647544ddedce851/src/Foundation/Exception/Handler.php#L181

Reply:
Is there anything in the base class that would suggest we can't just make that use throwable as well?

Reply:
I made some changes to the class, everything is now throwable oriented.